app/scripts/modules/core/src/search/infrastructure/SearchResultPods.tsx

```diff
@@ -59,6 +61,7 @@ export class SearchResultPods extends React.Component<ISearchResultPodsProps> {
                     key={category.category}
                     podData={category}
                     onRemoveItem={this.props.onRemoveItem}
+                    onResultClick={() => this.props.onResultClick(category.category)}
```

Comment:
Optional prop, but no checks if it exists. Add a default prop setting it to noop?

Reply:
even better, I made it a required prop